app/Infrastructure/Exceptions/UserNotFoundException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace App\Infrastructure\Exceptions;
+
+use Exception;
+
+class UserNotFoundException extends Exception
+{
```

Comment:
Here you can use a constructor and set a message and a status code... Research how 😄